mautic/mautic
# Update analytics.js to gtag to support ga4.

#### Description:

Google will turn off universal analytics on July 1st and the new GA4 codes do not work with analytics.js. This PR upgrades the analytics.js code to use the newer gtag code which will work with old UA codes and the newer GA4 codes.

This is a breaking change if the frontend uses analytics.js for goal or ecommerce tracking the end user would need to update that code to gtag or lose that reporting.

#### Steps to test this PR:

Create a new GA4 analytics account and get the code.
In mautic under Configuration > Tracking Settings > Google Analytics
Paste in your Google Analytics ID and enable on tracking page and landing page.
Load a landing page in a private tab.
Check analytics real time you will see your page hit.

PR for the 4.4 branch can be found here: https://github.com/mautic/mautic/pull/12525
